sdk/search/azure-search/src/main/java/com/azure/search/models/VisualFeature.java

```diff
@@ -13,15 +13,25 @@
  * Defines values for VisualFeature.
  */
 public enum VisualFeature {
```

Comment:
I think this is one Swagger version behind, because we recently removed two of these properties.

Reply:
Will generate again. Thanks!